backend/geonature/utils/utilssqlalchemy.py

```diff
@@ -502,7 +509,7 @@ def to_json_resp(
             filename="export_{}.{}".format(filename, extension),
         )
     return Response(
-        json.dumps(res, ensure_ascii=False, indent=indent),
+        json.dumps(res, ensure_ascii=False, indent=indent,default=additionnal_converter),
```

Comment:
Peut-tu préciser ton cas d'utilisation ?
Habituelement, tous les modèles sont pré-sérialisé grâce aux méthodes `as_dict()` ou `as_geofeature()` généré par les décorateur @serializable et @geoserializable
Ces décorateurs parsent les modèles en amont pour prégénérer la serialisation. ça évite de faire un if sur chaque ligne pour connaître la sérialisation.
En ajoutant cet `additionnal_converter` on refait un if sur chaque ligne, ce qui est un peu dommage...

Reply:
Le cas d'utilisation c'était une requête _plain SQL_ , où justement le décorateur ne s'applique pas.
Finalement je n'en ai plus l'usage car j'ai réussi à me dépatouiller avec une classe existante, mais la modif me paraissait intéressante. Ne peut-on pas avoir besoin d'utiliser du SQL brut pour des requêtes un peu complexes/coûteuses ?
Ma vision est peut-être biaisée car je suis plus efficace en SQL qu'avec l'ORM de sqlAlchemy (et j'imagine qu'on est plusieurs dans ce cas ?)

Reply:
Non tu as raison. Cela arrive de faire du SQL sur des requêtes complexes. 
Mais du coup rajouter ce IF sur toutes les routes (et chaque ligne !) peut être assez pénalisant, alors que dans 90% on a déjà notre résultat déjà sérialisé.
Je ferais un décorateur à part (pas dans utilssqlachemy par exemple)

Reply:
Je ne crois pas que la fonction  `additionnal_converter` soit appelée systématiquement, seulement sur là une une `TypeError` serait levée il me semble.

> If specified, default should be a function that gets called for objects that can’t otherwise be serialized. It should return a JSON encodable version of the object or raise a TypeError. If not specified, TypeError is raised. (https://docs.python.org/3/library/json.html#basic-usage)

Le décorateur @json_resp est bien pratique en tout cas :+1:

Reply:
Je viens de tester, tu as raison, j'avais mal compris la doc !
On intègre alors. On peut rajouter les UUID, float et autre type qui ne se sérialise pas tout seul